### PR TITLE
Fixed issue #16206: Slider tip indicates false error

### PR DESCRIPTION
**Pull request by gabrieljenik**
Added extra validation for situations where value got NaN

### CODE REVIEW

File: assets/packages/questions/numeric-slider/scripts/numeric-slider.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ var LSSlider = function (options) {
         },
         setValue = function (value) {
             value = value || parseFloat(position);
+            if (isNaN(value)) value = ""; // If value is NaN, validation fails while first moving the slider
```
Comment (thedirtypanda) [inline assets/packages/questions/numeric-slider/scripts/numeric-slider.js:101]: Please dont use short if clause. Use the one with {].

**Comment by gabrieljenik** [comment]
Reviewed

**Comment by sushmanadendlals** [comment]
Working as expected, Please refer Mantis for more details

**Comment by olleharstedt** [comment]
Thanks!

**Pull request merged**